### ISSUE TITLE
README.MD instructions not working to successfully run dev mode 

### ISSUE THREAD

**Issue by siddagra** [atomicptr/dauntless-builder-reforged#384]
I ran
`npm install -g yarn`
`yarn`
`yarn build:deps`
`yarn build:data`

node version is 18.

Opens up localhost:3000. Site is not rendered at all. The following errors show up in dev tools (inspect element):
```
inject.js:359     GET http://localhost:3000/favicon.ico 404 (Not Found)
extractIconBase64 @ inject.js:359
(anonymous) @ inject.js:323
(anonymous) @ inject.js:736
i18n.ts:134 i18next::backendConnector: loaded namespace translation for language en {alert: {…}, app-name: 'Dauntless Builder', components: {…}, drawer: {…}, feature-disabled-browser: 'This feature is currently disabled for this web browser.', …}
i18n.ts:134 i18next: languageChanged en-US
i18n.ts:134 i18next: initialized {debug: true, initImmediate: true, ns: Array(1), defaultNS: Array(1), fallbackLng: Array(1), …}
react-dom.development.js:86 Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Box3 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:4470:19)
    at LinkBox (http://localhost:3000/src/components/LinkBox.tsx:20:3)
    at p
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Typography2 (http://localhost:3000/node_modules/.vite/deps/@mui_material.js?v=a3eb7b1f:2893:22)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Grid2 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:6484:24)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Grid2 (http://localhost:3000/node_modules/.vite/deps/@mui_material.js?v=a3eb7b1f:12537:22)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Grid2 (http://localhost:3000/node_modules/.vite/deps/@mui_material.js?v=a3eb7b1f:12537:22)
    at Home (http://localhost:3000/src/pages/home/Home.tsx:36:7)
    at RenderedRoute (http://localhost:3000/node_modules/.vite/deps/react-router-dom.js?v=9afbfbd1:3133:5)
    at RenderedRoute (http://localhost:3000/node_modules/.vite/deps/react-router-dom.js?v=9afbfbd1:3133:5)
    at Routes (http://localhost:3000/node_modules/.vite/deps/react-router-dom.js?v=9afbfbd1:3542:5)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Container3 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:5702:19)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Box3 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:4470:19)
    at Layout (http://localhost:3000/src/components/Layout.tsx:38:3)
    at SnackbarProvider2 (http://localhost:3000/node_modules/.vite/deps/notistack.js?v=00acdd37:1227:24)
    at Router (http://localhost:3000/node_modules/.vite/deps/react-router-dom.js?v=9afbfbd1:3489:15)
    at BrowserRouter (http://localhost:3000/node_modules/.vite/deps/react-router-dom.js?v=9afbfbd1:3938:5)
    at ThemeProvider2 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:4801:5)
    at ThemeProvider3 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:4860:5)
    at ThemeProvider (http://localhost:3000/node_modules/.vite/deps/chunk-N6CLUAR5.js?v=c78af516:266:12)
    at DauntlessBuilderApp (http://localhost:3000/src/app.tsx:45:20)
    at Provider (http://localhost:3000/node_modules/.vite/deps/react-redux.js?v=f8a4da3c:979:3)
printWarning @ react-dom.development.js:86
error @ react-dom.development.js:60
validateDOMNesting @ react-dom.development.js:10849
createInstance @ react-dom.development.js:10930
completeWork @ react-dom.development.js:22187
completeUnitOfWork @ react-dom.development.js:26596
performUnitOfWork @ react-dom.development.js:26568
workLoopSync @ react-dom.development.js:26466
renderRootSync @ react-dom.development.js:26434
performConcurrentWorkOnRoot @ react-dom.development.js:25738
workLoop @ scheduler.development.js:266
flushWork @ scheduler.development.js:239
performWorkUntilDeadline @ scheduler.development.js:533
Show 13 more frames
BuildModel.ts:356 Uncaught TypeError: Cannot read properties of undefined (reading 'mods')
    at BuildModel.ts:356:59
    at e2.run (match.ts:162:12)
    at findPartByName (BuildModel.ts:365:10)
    at findPartInBuild (BuildModel.ts:395:12)
    at get partData [as partData] (BuildModel.ts:116:18)
    at get data [as data] (BuildModel.ts:90:25)
    at validateDoCellSlotsFitInTheirSlots (validate-do-cell-slots-fit-in-their-slots.ts:9:45)
    at validateBuild (validate-build.ts:20:17)
    at BuildModel.tryDeserialize (BuildModel.ts:233:16)
    at BuildCard (BuildCard.tsx:78:32)
(anonymous) @ BuildModel.ts:356
e2.run @ match.ts:162
findPartByName @ BuildModel.ts:365
findPartInBuild @ BuildModel.ts:395
get partData @ BuildModel.ts:116
get data @ BuildModel.ts:90
validateDoCellSlotsFitInTheirSlots @ validate-do-cell-slots-fit-in-their-slots.ts:9
validateBuild @ validate-build.ts:20
tryDeserialize @ BuildModel.ts:233
BuildCard @ BuildCard.tsx:78
renderWithHooks @ react-dom.development.js:16305
updateFunctionComponent @ react-dom.development.js:19588
updateSimpleMemoComponent @ react-dom.development.js:19425
updateMemoComponent @ react-dom.development.js:19284
beginWork @ react-dom.development.js:21673
callCallback2 @ react-dom.development.js:4164
invokeGuardedCallbackDev @ react-dom.development.js:4213
invokeGuardedCallback @ react-dom.development.js:4277
beginWork$1 @ react-dom.development.js:27451
performUnitOfWork @ react-dom.development.js:26557
workLoopSync @ react-dom.development.js:26466
renderRootSync @ react-dom.development.js:26434
performConcurrentWorkOnRoot @ react-dom.development.js:25738
workLoop @ scheduler.development.js:266
flushWork @ scheduler.development.js:239
performWorkUntilDeadline @ scheduler.development.js:533
Show 17 more frames
single-file-hooks-frames.js:1 SingleFile is hooking the IntersectionObserver API to detect and load deferred images.
y @ single-file-hooks-frames.js:1
e.IntersectionObserver @ single-file-hooks-frames.js:1
(anonymous) @ index.js:1
v2 @ index.js:1
constructClassInstance @ react-dom.development.js:14323
updateClassComponent @ react-dom.development.js:19688
beginWork @ react-dom.development.js:21611
beginWork$1 @ react-dom.development.js:27426
performUnitOfWork @ react-dom.development.js:26557
workLoopSync @ react-dom.development.js:26466
renderRootSync @ react-dom.development.js:26434
performConcurrentWorkOnRoot @ react-dom.development.js:25738
workLoop @ scheduler.development.js:266
flushWork @ scheduler.development.js:239
performWorkUntilDeadline @ scheduler.development.js:533
Show 13 more frames
BuildModel.ts:334 Uncaught TypeError: Cannot read properties of undefined (reading 'chambers')
    at BuildModel.ts:334:60
    at e2.run (match.ts:162:12)
    at findPartByName (BuildModel.ts:365:10)
    at findPartInBuild (BuildModel.ts:395:12)
    at get partData [as partData] (BuildModel.ts:109:26)
    at get data [as data] (BuildModel.ts:90:25)
    at validateDoCellSlotsFitInTheirSlots (validate-do-cell-slots-fit-in-their-slots.ts:9:45)
    at validateBuild (validate-build.ts:20:17)
    at BuildModel.tryDeserialize (BuildModel.ts:233:16)
    at BuildCard (BuildCard.tsx:78:32)
(anonymous) @ BuildModel.ts:334
e2.run @ match.ts:162
findPartByName @ BuildModel.ts:365
findPartInBuild @ BuildModel.ts:395
get partData @ BuildModel.ts:109
get data @ BuildModel.ts:90
validateDoCellSlotsFitInTheirSlots @ validate-do-cell-slots-fit-in-their-slots.ts:9
validateBuild @ validate-build.ts:20
tryDeserialize @ BuildModel.ts:233
BuildCard @ BuildCard.tsx:78
renderWithHooks @ react-dom.development.js:16305
updateFunctionComponent @ react-dom.development.js:19588
updateSimpleMemoComponent @ react-dom.development.js:19425
updateMemoComponent @ react-dom.development.js:19284
beginWork @ react-dom.development.js:21673
callCallback2 @ react-dom.development.js:4164
invokeGuardedCallbackDev @ react-dom.development.js:4213
invokeGuardedCallback @ react-dom.development.js:4277
beginWork$1 @ react-dom.development.js:27451
performUnitOfWork @ react-dom.development.js:26557
workLoopSync @ react-dom.development.js:26466
renderRootSync @ react-dom.development.js:26434
performConcurrentWorkOnRoot @ react-dom.development.js:25738
workLoop @ scheduler.development.js:266
flushWork @ scheduler.development.js:239
performWorkUntilDeadline @ scheduler.development.js:533
Show 17 more frames
BuildModel.ts:308 Uncaught TypeError: Cannot read properties of undefined (reading 'mods')
    at BuildModel.ts:308:63
    at e2.run (match.ts:162:12)
    at findPartByName (BuildModel.ts:365:10)
    at findPartInBuild (BuildModel.ts:395:12)
    at get partData [as partData] (BuildModel.ts:116:18)
    at get data [as data] (BuildModel.ts:90:25)
    at validateDoCellSlotsFitInTheirSlots (validate-do-cell-slots-fit-in-their-slots.ts:9:45)
    at validateBuild (validate-build.ts:20:17)
    at BuildModel.tryDeserialize (BuildModel.ts:233:16)
    at BuildCard (BuildCard.tsx:78:32)
(anonymous) @ BuildModel.ts:308
e2.run @ match.ts:162
findPartByName @ BuildModel.ts:365
findPartInBuild @ BuildModel.ts:395
get partData @ BuildModel.ts:116
get data @ BuildModel.ts:90
validateDoCellSlotsFitInTheirSlots @ validate-do-cell-slots-fit-in-their-slots.ts:9
validateBuild @ validate-build.ts:20
tryDeserialize @ BuildModel.ts:233
BuildCard @ BuildCard.tsx:78
renderWithHooks @ react-dom.development.js:16305
updateFunctionComponent @ react-dom.development.js:19588
updateSimpleMemoComponent @ react-dom.development.js:19425
updateMemoComponent @ react-dom.development.js:19284
beginWork @ react-dom.development.js:21673
callCallback2 @ react-dom.development.js:4164
invokeGuardedCallbackDev @ react-dom.development.js:4213
invokeGuardedCallback @ react-dom.development.js:4277
beginWork$1 @ react-dom.development.js:27451
performUnitOfWork @ react-dom.development.js:26557
workLoopSync @ react-dom.development.js:26466
renderRootSync @ react-dom.development.js:26434
performConcurrentWorkOnRoot @ react-dom.development.js:25738
workLoop @ scheduler.development.js:266
flushWork @ scheduler.development.js:239
performWorkUntilDeadline @ scheduler.development.js:533
Show 17 more frames
BuildModel.ts:356 Uncaught TypeError: Cannot read properties of undefined (reading 'mods')
    at BuildModel.ts:356:59
    at e2.run (match.ts:162:12)
    at findPartByName (BuildModel.ts:365:10)
    at findPartInBuild (BuildModel.ts:395:12)
    at get partData [as partData] (BuildModel.ts:116:18)
    at get data [as data] (BuildModel.ts:90:25)
    at validateDoCellSlotsFitInTheirSlots (validate-do-cell-slots-fit-in-their-slots.ts:9:45)
    at validateBuild (validate-build.ts:20:17)
    at BuildModel.tryDeserialize (BuildModel.ts:233:16)
    at BuildCard (BuildCard.tsx:78:32)
(anonymous) @ BuildModel.ts:356
e2.run @ match.ts:162
findPartByName @ BuildModel.ts:365
findPartInBuild @ BuildModel.ts:395
get partData @ BuildModel.ts:116
get data @ BuildModel.ts:90
validateDoCellSlotsFitInTheirSlots @ validate-do-cell-slots-fit-in-their-slots.ts:9
validateBuild @ validate-build.ts:20
tryDeserialize @ BuildModel.ts:233
BuildCard @ BuildCard.tsx:78
renderWithHooks @ react-dom.development.js:16305
updateFunctionComponent @ react-dom.development.js:19588
updateSimpleMemoComponent @ react-dom.development.js:19425
updateMemoComponent @ react-dom.development.js:19284
beginWork @ react-dom.development.js:21673
callCallback2 @ react-dom.development.js:4164
invokeGuardedCallbackDev @ react-dom.development.js:4213
invokeGuardedCallback @ react-dom.development.js:4277
beginWork$1 @ react-dom.development.js:27451
performUnitOfWork @ react-dom.development.js:26557
workLoopSync @ react-dom.development.js:26466
renderRootSync @ react-dom.development.js:26434
recoverFromConcurrentError @ react-dom.development.js:25850
performConcurrentWorkOnRoot @ react-dom.development.js:25750
workLoop @ scheduler.development.js:266
flushWork @ scheduler.development.js:239
performWorkUntilDeadline @ scheduler.development.js:533
Show 18 more frames
BuildModel.ts:334 Uncaught TypeError: Cannot read properties of undefined (reading 'chambers')
    at BuildModel.ts:334:60
    at e2.run (match.ts:162:12)
    at findPartByName (BuildModel.ts:365:10)
    at findPartInBuild (BuildModel.ts:395:12)
    at get partData [as partData] (BuildModel.ts:109:26)
    at get data [as data] (BuildModel.ts:90:25)
    at validateDoCellSlotsFitInTheirSlots (validate-do-cell-slots-fit-in-their-slots.ts:9:45)
    at validateBuild (validate-build.ts:20:17)
    at BuildModel.tryDeserialize (BuildModel.ts:233:16)
    at BuildCard (BuildCard.tsx:78:32)
(anonymous) @ BuildModel.ts:334
e2.run @ match.ts:162
findPartByName @ BuildModel.ts:365
findPartInBuild @ BuildModel.ts:395
get partData @ BuildModel.ts:109
get data @ BuildModel.ts:90
validateDoCellSlotsFitInTheirSlots @ validate-do-cell-slots-fit-in-their-slots.ts:9
validateBuild @ validate-build.ts:20
tryDeserialize @ BuildModel.ts:233
BuildCard @ BuildCard.tsx:78
renderWithHooks @ react-dom.development.js:16305
updateFunctionComponent @ react-dom.development.js:19588
updateSimpleMemoComponent @ react-dom.development.js:19425
updateMemoComponent @ react-dom.development.js:19284
beginWork @ react-dom.development.js:21673
callCallback2 @ react-dom.development.js:4164
invokeGuardedCallbackDev @ react-dom.development.js:4213
invokeGuardedCallback @ react-dom.development.js:4277
beginWork$1 @ react-dom.development.js:27451
performUnitOfWork @ react-dom.development.js:26557
workLoopSync @ react-dom.development.js:26466
renderRootSync @ react-dom.development.js:26434
recoverFromConcurrentError @ react-dom.development.js:25850
performConcurrentWorkOnRoot @ react-dom.development.js:25750
workLoop @ scheduler.development.js:266
flushWork @ scheduler.development.js:239
performWorkUntilDeadline @ scheduler.development.js:533
Show 18 more frames
BuildModel.ts:308 Uncaught TypeError: Cannot read properties of undefined (reading 'mods')
    at BuildModel.ts:308:63
    at e2.run (match.ts:162:12)
    at findPartByName (BuildModel.ts:365:10)
    at findPartInBuild (BuildModel.ts:395:12)
    at get partData [as partData] (BuildModel.ts:116:18)
    at get data [as data] (BuildModel.ts:90:25)
    at validateDoCellSlotsFitInTheirSlots (validate-do-cell-slots-fit-in-their-slots.ts:9:45)
    at validateBuild (validate-build.ts:20:17)
    at BuildModel.tryDeserialize (BuildModel.ts:233:16)
    at BuildCard (BuildCard.tsx:78:32)
(anonymous) @ BuildModel.ts:308
e2.run @ match.ts:162
findPartByName @ BuildModel.ts:365
findPartInBuild @ BuildModel.ts:395
get partData @ BuildModel.ts:116
get data @ BuildModel.ts:90
validateDoCellSlotsFitInTheirSlots @ validate-do-cell-slots-fit-in-their-slots.ts:9
validateBuild @ validate-build.ts:20
tryDeserialize @ BuildModel.ts:233
BuildCard @ BuildCard.tsx:78
renderWithHooks @ react-dom.development.js:16305
updateFunctionComponent @ react-dom.development.js:19588
updateSimpleMemoComponent @ react-dom.development.js:19425
updateMemoComponent @ react-dom.development.js:19284
beginWork @ react-dom.development.js:21673
callCallback2 @ react-dom.development.js:4164
invokeGuardedCallbackDev @ react-dom.development.js:4213
invokeGuardedCallback @ react-dom.development.js:4277
beginWork$1 @ react-dom.development.js:27451
performUnitOfWork @ react-dom.development.js:26557
workLoopSync @ react-dom.development.js:26466
renderRootSync @ react-dom.development.js:26434
recoverFromConcurrentError @ react-dom.development.js:25850
performConcurrentWorkOnRoot @ react-dom.development.js:25750
workLoop @ scheduler.development.js:266
flushWork @ scheduler.development.js:239
performWorkUntilDeadline @ scheduler.development.js:533
Show 18 more frames
react-dom.development.js:86 Warning: Using UNSAFE_componentWillMount in strict mode is not recommended and may indicate bugs in your code. See https://reactjs.org/link/unsafe-component-lifecycles for details.

* Move code with side effects to componentDidMount, and set initial state in the constructor.

Please update the following components: SideEffect(NullComponent2)
printWarning @ react-dom.development.js:86
error @ react-dom.development.js:60
ReactStrictModeWarnings.flushPendingUnsafeLifecycleWarnings @ react-dom.development.js:12888
flushRenderPhaseStrictModeWarningsInDEV @ react-dom.development.js:27310
commitRootImpl @ react-dom.development.js:26702
commitRoot @ react-dom.development.js:26682
finishConcurrentRender @ react-dom.development.js:25892
performConcurrentWorkOnRoot @ react-dom.development.js:25809
workLoop @ scheduler.development.js:266
flushWork @ scheduler.development.js:239
performWorkUntilDeadline @ scheduler.development.js:533
Show 11 more frames
react-dom.development.js:18687 The above error occurred in the <BuildCard> component:

    at BuildCard (http://localhost:3000/src/components/BuildCard.tsx:58:3)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Grid2 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:6484:24)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Grid2 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:6484:24)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Grid2 (http://localhost:3000/node_modules/.vite/deps/@mui_material.js?v=a3eb7b1f:12537:22)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Grid2 (http://localhost:3000/node_modules/.vite/deps/@mui_material.js?v=a3eb7b1f:12537:22)
    at Home (http://localhost:3000/src/pages/home/Home.tsx:36:7)
    at RenderedRoute (http://localhost:3000/node_modules/.vite/deps/react-router-dom.js?v=9afbfbd1:3133:5)
    at RenderedRoute (http://localhost:3000/node_modules/.vite/deps/react-router-dom.js?v=9afbfbd1:3133:5)
    at Routes (http://localhost:3000/node_modules/.vite/deps/react-router-dom.js?v=9afbfbd1:3542:5)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Container3 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:5702:19)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Box3 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:4470:19)
    at Layout (http://localhost:3000/src/components/Layout.tsx:38:3)
    at SnackbarProvider2 (http://localhost:3000/node_modules/.vite/deps/notistack.js?v=00acdd37:1227:24)
    at Router (http://localhost:3000/node_modules/.vite/deps/react-router-dom.js?v=9afbfbd1:3489:15)
    at BrowserRouter (http://localhost:3000/node_modules/.vite/deps/react-router-dom.js?v=9afbfbd1:3938:5)
    at ThemeProvider2 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:4801:5)
    at ThemeProvider3 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:4860:5)
    at ThemeProvider (http://localhost:3000/node_modules/.vite/deps/chunk-N6CLUAR5.js?v=c78af516:266:12)
    at DauntlessBuilderApp (http://localhost:3000/src/app.tsx:45:20)
    at Provider (http://localhost:3000/node_modules/.vite/deps/react-redux.js?v=f8a4da3c:979:3)

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://reactjs.org/link/error-boundaries to learn more about error boundaries.
logCapturedError @ react-dom.development.js:18687
update.callback @ react-dom.development.js:18720
callCallback @ react-dom.development.js:13923
commitUpdateQueue @ react-dom.development.js:13944
commitLayoutEffectOnFiber @ react-dom.development.js:23391
commitLayoutMountEffects_complete @ react-dom.development.js:24688
commitLayoutEffects_begin @ react-dom.development.js:24674
commitLayoutEffects @ react-dom.development.js:24612
commitRootImpl @ react-dom.development.js:26823
commitRoot @ react-dom.development.js:26682
finishConcurrentRender @ react-dom.development.js:25892
performConcurrentWorkOnRoot @ react-dom.development.js:25809
workLoop @ scheduler.development.js:266
flushWork @ scheduler.development.js:239
performWorkUntilDeadline @ scheduler.development.js:533
Show 15 more frames
react-dom.development.js:18687 The above error occurred in the <BuildCard> component:

    at BuildCard (http://localhost:3000/src/components/BuildCard.tsx:58:3)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Grid2 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:6484:24)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Grid2 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:6484:24)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Grid2 (http://localhost:3000/node_modules/.vite/deps/@mui_material.js?v=a3eb7b1f:12537:22)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Grid2 (http://localhost:3000/node_modules/.vite/deps/@mui_material.js?v=a3eb7b1f:12537:22)
    at Home (http://localhost:3000/src/pages/home/Home.tsx:36:7)
    at RenderedRoute (http://localhost:3000/node_modules/.vite/deps/react-router-dom.js?v=9afbfbd1:3133:5)
    at RenderedRoute (http://localhost:3000/node_modules/.vite/deps/react-router-dom.js?v=9afbfbd1:3133:5)
    at Routes (http://localhost:3000/node_modules/.vite/deps/react-router-dom.js?v=9afbfbd1:3542:5)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Container3 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:5702:19)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Box3 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:4470:19)
    at Layout (http://localhost:3000/src/components/Layout.tsx:38:3)
    at SnackbarProvider2 (http://localhost:3000/node_modules/.vite/deps/notistack.js?v=00acdd37:1227:24)
    at Router (http://localhost:3000/node_modules/.vite/deps/react-router-dom.js?v=9afbfbd1:3489:15)
    at BrowserRouter (http://localhost:3000/node_modules/.vite/deps/react-router-dom.js?v=9afbfbd1:3938:5)
    at ThemeProvider2 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:4801:5)
    at ThemeProvider3 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:4860:5)
    at ThemeProvider (http://localhost:3000/node_modules/.vite/deps/chunk-N6CLUAR5.js?v=c78af516:266:12)
    at DauntlessBuilderApp (http://localhost:3000/src/app.tsx:45:20)
    at Provider (http://localhost:3000/node_modules/.vite/deps/react-redux.js?v=f8a4da3c:979:3)

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://reactjs.org/link/error-boundaries to learn more about error boundaries.
logCapturedError @ react-dom.development.js:18687
update.callback @ react-dom.development.js:18720
callCallback @ react-dom.development.js:13923
commitUpdateQueue @ react-dom.development.js:13944
commitLayoutEffectOnFiber @ react-dom.development.js:23391
commitLayoutMountEffects_complete @ react-dom.development.js:24688
commitLayoutEffects_begin @ react-dom.development.js:24674
commitLayoutEffects @ react-dom.development.js:24612
commitRootImpl @ react-dom.development.js:26823
commitRoot @ react-dom.development.js:26682
finishConcurrentRender @ react-dom.development.js:25892
performConcurrentWorkOnRoot @ react-dom.development.js:25809
workLoop @ scheduler.development.js:266
flushWork @ scheduler.development.js:239
performWorkUntilDeadline @ scheduler.development.js:533
Show 15 more frames
react-dom.development.js:18687 The above error occurred in the <BuildCard> component:

    at BuildCard (http://localhost:3000/src/components/BuildCard.tsx:58:3)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Grid2 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:6484:24)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Grid2 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:6484:24)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Grid2 (http://localhost:3000/node_modules/.vite/deps/@mui_material.js?v=a3eb7b1f:12537:22)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Grid2 (http://localhost:3000/node_modules/.vite/deps/@mui_material.js?v=a3eb7b1f:12537:22)
    at Home (http://localhost:3000/src/pages/home/Home.tsx:36:7)
    at RenderedRoute (http://localhost:3000/node_modules/.vite/deps/react-router-dom.js?v=9afbfbd1:3133:5)
    at RenderedRoute (http://localhost:3000/node_modules/.vite/deps/react-router-dom.js?v=9afbfbd1:3133:5)
    at Routes (http://localhost:3000/node_modules/.vite/deps/react-router-dom.js?v=9afbfbd1:3542:5)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Container3 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:5702:19)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:2022:46
    at Box3 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:4470:19)
    at Layout (http://localhost:3000/src/components/Layout.tsx:38:3)
    at SnackbarProvider2 (http://localhost:3000/node_modules/.vite/deps/notistack.js?v=00acdd37:1227:24)
    at Router (http://localhost:3000/node_modules/.vite/deps/react-router-dom.js?v=9afbfbd1:3489:15)
    at BrowserRouter (http://localhost:3000/node_modules/.vite/deps/react-router-dom.js?v=9afbfbd1:3938:5)
    at ThemeProvider2 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:4801:5)
    at ThemeProvider3 (http://localhost:3000/node_modules/.vite/deps/chunk-6LZUM6YC.js?v=c78af516:4860:5)
    at ThemeProvider (http://localhost:3000/node_modules/.vite/deps/chunk-N6CLUAR5.js?v=c78af516:266:12)
    at DauntlessBuilderApp (http://localhost:3000/src/app.tsx:45:20)
    at Provider (http://localhost:3000/node_modules/.vite/deps/react-redux.js?v=f8a4da3c:979:3)

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://reactjs.org/link/error-boundaries to learn more about error boundaries.
logCapturedError @ react-dom.development.js:18687
update.callback @ react-dom.development.js:18720
callCallback @ react-dom.development.js:13923
commitUpdateQueue @ react-dom.development.js:13944
commitLayoutEffectOnFiber @ react-dom.development.js:23391
commitLayoutMountEffects_complete @ react-dom.development.js:24688
commitLayoutEffects_begin @ react-dom.development.js:24674
commitLayoutEffects @ react-dom.development.js:24612
commitRootImpl @ react-dom.development.js:26823
commitRoot @ react-dom.development.js:26682
finishConcurrentRender @ react-dom.development.js:25892
performConcurrentWorkOnRoot @ react-dom.development.js:25809
workLoop @ scheduler.development.js:266
flushWork @ scheduler.development.js:239
performWorkUntilDeadline @ scheduler.development.js:533
Show 15 more frames
react-dom.development.js:26923 Uncaught TypeError: Cannot read properties of undefined (reading 'mods')
    at BuildModel.ts:356:59
    at e2.run (match.ts:162:12)
    at findPartByName (BuildModel.ts:365:10)
    at findPartInBuild (BuildModel.ts:395:12)
    at get partData [as partData] (BuildModel.ts:116:18)
    at get data [as data] (BuildModel.ts:90:25)
    at validateDoCellSlotsFitInTheirSlots (validate-do-cell-slots-fit-in-their-slots.ts:9:45)
    at validateBuild (validate-build.ts:20:17)
    at BuildModel.tryDeserialize (BuildModel.ts:233:16)
    at BuildCard (BuildCard.tsx:78:32)
(anonymous) @ BuildModel.ts:356
e2.run @ match.ts:162
findPartByName @ BuildModel.ts:365
findPartInBuild @ BuildModel.ts:395
get partData @ BuildModel.ts:116
get data @ BuildModel.ts:90
validateDoCellSlotsFitInTheirSlots @ validate-do-cell-slots-fit-in-their-slots.ts:9
validateBuild @ validate-build.ts:20
tryDeserialize @ BuildModel.ts:233
BuildCard @ BuildCard.tsx:78
renderWithHooks @ react-dom.development.js:16305
updateFunctionComponent @ react-dom.development.js:19588
updateSimpleMemoComponent @ react-dom.development.js:19425
updateMemoComponent @ react-dom.development.js:19284
beginWork @ react-dom.development.js:21673
beginWork$1 @ react-dom.development.js:27426
performUnitOfWork @ react-dom.development.js:26557
workLoopSync @ react-dom.development.js:26466
renderRootSync @ react-dom.development.js:26434
recoverFromConcurrentError @ react-dom.development.js:25850
performConcurrentWorkOnRoot @ react-dom.development.js:25750
workLoop @ scheduler.development.js:266
flushWork @ scheduler.development.js:239
performWorkUntilDeadline @ scheduler.development.js:533
Show 15 more frames
inject.js:359     GET http://localhost:3000/favicon.ico 404 (Not Found)
extractIconBase64 @ inject.js:359
setTimeout (async)
(anonymous) @ inject.js:325
load (async)
(anonymous) @ inject.js:324
(anonymous) @ inject.js:736

```

**Comment by atomicptr** [comment]
I tried setting up a new project in a VM (Ubuntu 23.04 / Node 18) and it just worked fine. You sure you ran the build:data thing because the errors look a bit like there is no data (again, like the last time). Please check the contents of src/json/data.json and post that here and also whatever build:data outputs :)

@ds-exe can you check Windows?

**Comment by ds-exe** [comment]
Fixed with #387
